### ISSUE TITLE
Bug when update to Laravel v5.0.21.

### ISSUE THREAD

**Issue by jonathanpmartins** [LaravelCollective/annotations#18]
When I update to Laravel 5.0.21 the command 'php artisan route:scan' and 'php artisan route:list' dont work anymore. I get this error:

```
[Symfony\Component\Debug\Exception\FatalErrorException]  
    Call to a member function routeScans() on a non-object 
```

**Comment by chood531** [comment]
yes, same error for me too.

**Comment by tshafer** [comment]
Adam or I will take a look soon.

—
Tom Shafer

**Comment by adamgoose** [comment]
Fixed in v5.0.5

**Comment by jonathanpmartins** [comment]
Sorry, I'm using version 5.0.4. Updating now!
